earth_enterprise/docker/README.md

```diff
@@ -0,0 +1,182 @@
+# Docker Image for Running GE Open Source Earth Server and Fusion Server
+
+## Setting up Docker
+
+### Ubuntu
+
+Install Docker as root:
+
+```BASH
+# apt-get install docker.io
+```
+
+Add users that you want to have access to Docker to the `docker` group:
+
+```BASH
+# adduser muser docker
+```
+
+
+## Pulling an Image from Docker Cloud
+
+You can get a pre-built image from a Docker registry.
+
+```BASH
+docker pull thermopylae/opengee-experimental
+```
+
+
+## Running the Container
+
+### Running the Open GEE Servers
+
+#### Running Open GEE Server with Persistent Globe and Map Databases
+
+In order to be able to build and serve Earth databases that persist between 
+instantiations of the Docker image, you need a persistent assets directory 
+outside of the Docker container.  Another alternative is to run a container
+with a temporary assets directory, and commit the running to container to a
+new image after you've build your assets.
+
+To run the Docker container using a persistent globe assets directory outside 
+the container, use the following script.  Set the `HOST_GEVOL_PATH`
+environment variable to the path outside the container where you want to store
+your persistent assets.
+
+```BASH
+HOST_GEVOL_PATH=/persistent/gevol/ ./bin/start-docker-image.sh
+```
+
+You can optionally set the `GEE_SERVER_PORT` and even  `IMAGE_TAG` to override
```

Comment:
Minor: maybe we should change the default server port. It seems more likely than not that someone would already have a server on port 80